src/test/java/com/amihaiemil/docker/RtPluginTestCase.java

```diff
@@ -106,4 +110,158 @@ public void inspectsItself() throws Exception {
         );
     }
 
+    /**
+     * RtPlugin enable itself.
+     * @throws Exception If something goes wrong.
+     */
+    @Ignore
+    @Test
+    public void enablesItself() throws Exception {
+        new ListedPlugins(
+            new AssertRequest(
+                new Response(
+                    HttpStatus.SC_NO_CONTENT
+                )
+            ),
+            URI.create("http://localhost/plugins"),
+            DOCKER
+        ).pullAndInstall(
+            "vieus/sshfs",
+            "sshfs",
+            Json.createArrayBuilder().add(
+                Json.createObjectBuilder()
+                .add("Name", "network")
+                .add("Description", "")
+                .add("Value", "host")
+            ).build()
+        );
+        final Plugin plugin = new RtPlugin(
+            Json.createObjectBuilder().build(),
+            new AssertRequest(
+                new Response(
+                    HttpStatus.SC_OK
+                ),
+                new Condition(
+                    "Method should be a POST",
+                    req -> "POST".equals(req.getRequestLine().getMethod())
+                ),
+                new Condition(
+                    "Resource path must be /{name}/enable",
+                    req -> req.getRequestLine().getUri()
+                    .endsWith("/sshfs/enable")
+                )
+            ),
+            URI.create("http://localhost/plugins/sshfs/enable"),
```

Comment:
@paulodamaso URI of the RtPlugin should end with name. `/enable` is something that will be added by calling the `enable() `method. So please remove` /enable` from the end.